File: javaspring/src/main/java/patterns/sample/utils/contentnegotiation/models/Weather.java
package patterns.sample.utils.contentnegotiation.models;

import java.time.LocalDate;

public class Weather {
    private LocalDate dateTime;
    private double temperature;

    public Weather(LocalDate dateTime, double temperature) {
        this.dateTime = dateTime;
        this.temperature = temperature;
    }

    public double getTemperature() {
        return temperature;
    }

    public void setTemperature(double temperature) {
        this.temperature = temperature;
    }

    public LocalDate getDateTime() {
        return dateTime;
    }

    public void setDateTime(LocalDate dateTime) {
        this.dateTime = dateTime;
    }
}

File: dotnet/Utils/ContentNegotiation/Models/Weather.cs
using System;

namespace sampleApi.Utils.ContentNegotiation.Models
{
    public class Weather
    {
        private DateTime _date;
        private double _temperature;

        public Weather(DateTime date, double temperature)
        {
            this._date = date;
            this._temperature = temperature;
        }

        public Weather() { }

        public DateTime Date { get; set; }
        public double Temperature { get; set; }
    }
}